### ISSUE TITLE
Add Arrays to Questions

### ISSUE THREAD

**Issue by brittdawn** [brittdawn/about_me#15]
For the five yes/no questions, your questions, the correct answers, and the two possible responses should be moved into three separate arrays. (The same applies if you've added in any other pieces to the questions besides the most basic structure.) Once this is done you can replace that content in the functions by accessing the arrays. If you have already done this, but with a similar architecture, that is fine. a-c-p
